agenta-web/src/components/TestSetTable/TestsetTable.tsx

```diff
@@ -203,18 +203,11 @@ const TestsetTable: React.FC<testsetTableProps> = ({mode}) => {
 
                 setRowData(separateRowData)
             }
-            setInputValues(
-                newColDefs.filter((col) => !!col.field).map((col) => col.field),
-                () => {
-                    setTimeout(() => {
```

Comment:
@MohammedMaaz do you know why do we have this setTimeout and is it fine to remove it ?